src/services/Content/ContentQueries.ts

```diff
@@ -79,3 +84,9 @@ export const useDeleteContentItemMutate = (
     },
   });
 };
+
+export const useRepositoryParams = () =>
+  useQuery<RepositoryParamsResponse>(REPOSITORY_PARAMS_KEY, getRepositoryParams, {
+    keepPreviousData: true,
+    staleTime: Infinity,
+  });
```

Comment:
We need to mark the datas staletime as: Infinity... this will ensure we don't refetch it.